src/nxt_main_process.c

```diff
@@ -1433,3 +1486,106 @@ nxt_main_port_access_log_handler(nxt_task_t *task, nxt_port_recv_msg_t *msg)
                                      msg->port_msg.stream, 0, NULL);
     }
 }
+
+static nxt_int_t 
+nxt_init_set_isolation(nxt_task_t *task, nxt_process_init_t *init, nxt_conf_value_t *isolation)
+{
+    nxt_conf_value_t *object;
+    nxt_str_t        nsname  = nxt_string("namespaces");
+    nxt_str_t        uidname = nxt_string("uidmap");
+    nxt_str_t        gidname = nxt_string("gidmap");
+    nxt_int_t        ret;
+
+    if (isolation == NULL) {
+        return NXT_OK;
+    }
+
+    object = nxt_conf_get_object_member(isolation, &nsname, NULL);
+    if (object != NULL) {
+        ret = nxt_init_set_ns(task, init, object);
+        if (ret != NXT_OK) {
+            return ret;
+        }
+    }
+
+    object = nxt_conf_get_object_member(isolation, &uidname, NULL);
+    if (object != NULL) {
+        init->isolation.clone.uidmap = object;
+    }
+
+    object = nxt_conf_get_object_member(isolation, &gidname, NULL);
+    if (object != NULL) {
+        init->isolation.clone.gidmap = object;
+    }
+
+    return NXT_OK;    
+}
+
+static nxt_int_t 
+nxt_init_set_ns(nxt_task_t *task, nxt_process_init_t *init, nxt_conf_value_t *namespaces)
+{
+    nxt_conf_value_t *value;
+    nxt_str_t        name;
+    uint32_t         index;
+    nxt_int_t        flag = 0;
+    
+    index = 0;
+    while ((value = nxt_conf_next_object_member(namespaces, &name, &index)) != NULL) {
+        flag = 0;
+
+        if (nxt_slow_path(nxt_conf_type(value) != NXT_CONF_BOOLEAN)) {
+            nxt_alert(task, "unexpected namespace value: \"%V\". "
```

Comment:
%V expects nxt_str_t *, not nxt_conf_value_t *

Also, I don't think this check is needed as nothing other than boolean can pass the validation.